core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -91,15 +91,15 @@
           </a>
           <ul ngbDropdownMenu
               [ngStyle]="{'left.px': learnDropdownOffset}"
-              [ngClass]="{'classroom-disabled': !CLASSROOM_PROMOS_ARE_ENABLED, 'classroom-enabled': CLASSROOM_PROMOS_ARE_ENABLED}"
+              [ngClass]="'classroom-enabled'"
```

Comment:
Just add classroom-enabled to the class attr below

Reply:
Updated